### DISCUSSION TITLE
I am  a realy student

### DISCUSSION THREAD

**Discussion by kawuwa** [community/community#33941] [GitHub Education]
I'm really a student. I've applied for **dozens of times**. Why don't you pass me?

**Comment by james-villanueva-freee** [comment]
Hi @kawuwa. I was still also a student (from a bootcamp) until last year when I landed my first job.
I only have a 6 months of experience coding, but in those 6 months, I tried creating small projects to showcase in my portfolio.
If you currently don't have one, it would significantly increase your chances since you can show something that you built yourself to your interviewer. It's also helpful to try to ask the interviewer who failed you what areas do they think you are lacking so that so you can work on them. Don't lose hope. Even if you receive dozens of rejections, always remember you only have to get a single YES. Good luck! 🙇🏼‍♂️

**Comment by kawuwa** [comment]
Thank you very much for your encouragement! I'm excited that my application has finally passed!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月23日(星期五) 晚上10:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] I am  a realy student (Discussion #33941)





 
Hi @kawuwa. I was still also a student (from a bootcamp) until last year when I landed my first job.
 I only have a 6 months of experience coding, but in those 6 months, I tried creating small projects to showcase in my portfolio.
 If you currently don't have one, it would significantly increase your chances since you can show something that you built yourself to your interviewer. It's also helpful to try to ask the interviewer who failed you what areas do they think you are lacking so that so you can work on them. Don't lose hope. Even if you receive dozens of rejections, always remember you only have to get a single YES. Good luck! 🙇🏼‍♂️
